qbo_face_following:
  ros__parameters:
    move_base: false
    move_head: true
    send_stop: false

    search_pan_vel: 0.6
    search_pan_step: 0.2
    search_min_pan: -1.0
    search_max_pan: 1.0

    search_tilt_vel: 0.6
    search_tilt_levels: [-0.5, 0.0, 0.3]
    search_min_tilt: -0.5
    search_max_tilt: 0.3

    scan_interval: 2.0
    search_pause_duration: 1.0

    desired_distance: 1.0

